airflow/operators/bash.py

```diff
@@ -134,13 +137,20 @@ def __init__(
         env: Optional[Dict[str, str]] = None,
         output_encoding: str = 'utf-8',
         skip_exit_code: int = 99,
+        cwd: str = None,
         **kwargs,
     ) -> None:
         super().__init__(**kwargs)
         self.bash_command = bash_command
         self.env = env
         self.output_encoding = output_encoding
         self.skip_exit_code = skip_exit_code
+        self.cwd = cwd
+        if self.cwd is not None:
+            if not os.path.exists(self.cwd):
+                raise AirflowException(f"Can not find the cwd: {cwd}")
+            if not os.path.isdir(self.cwd):
+                raise AirflowException(f"The cwd {cwd} must be a directory")
```

Comment:
This would raise an error on DAG parsing and force the scheduler to have the same directory set up as the task runner (which can be different machines). So I think we should *not* check `self.cwd` here, but only in `execute()`.

Reply:
You are right. And the unit test already did cover it.
I will update it

Reply:
I've updated it.

Reply:
I didn't saw you commit the changes.
But I think I made the same changes as yours.